iron-focusables-helper.html

```diff
@@ -127,18 +127,17 @@
         //  </div>
         // TODO(valdrin) support ShadowDOM v1 when upgrading to Polymer v2.0.
         var children;
-        if (element.localName === 'content') {
+        if (element.localName === 'content' || element.localName === 'slot') {
```

Comment:
We cannot, as this method will search deeply in the shadow roots of the elements; in Polymer 1.x there might be elements that are still using `<content>` - e.g. some element not updated yet to hybrid (user created Polymer Element)